Assets/MRTK/Providers/WindowsMixedReality/Shared/Editor/WindowsMixedRealityCameraSettingsProfileInspector.cs

```diff
@@ -46,7 +46,7 @@ public override void OnInspectorGUI()
                     EditorGUILayout.LabelField("Mixed Reality Capture Settings (Experimental)", EditorStyles.boldLabel);
                     InspectorUIUtility.RenderDocumentationButton(MRCDocURL);
                 }
-                EditorGUILayout.HelpBox("Render from PV Camera is supported on Unity 2018.4.13f1 or newer and 2019.3.0f1 or newer. Enabling the feature on other versions may result in incorrect capture behavior.", MessageType.Info);
+                EditorGUILayout.HelpBox("Render from PC camera is supported in Unity 2018.4.13 and newer if using Unity 2018, and Unity 2019.4.9f1 and newer if using Unity 2019. Enabling the feature on other versions may result in incorrect capture behavior.", MessageType.Info);
```

Comment:
```suggestion
                EditorGUILayout.HelpBox("Render from PV camera is supported in Unity 2018.4.13 and newer if using Unity 2018, and Unity 2019.4.9f1 and newer if using Unity 2019. Enabling the feature on other versions may result in incorrect capture behavior.", MessageType.Info);
```

Reply:
Do we also want to provide a clarification on legacy XR vs XR SDK?

Reply:
The only thing we need to add is that it doesn't work with XR SDK on Unity 2019 right? Does this text then make sense?

"Render from PV camera is supported in Unity 2018.4.13 and newer if using Unity 2018, and Unity 2019.4.9f1 and newer if using Unity 2019. **It does not work in Unity 2019 if using XR SDK.** Enabling the feature on other versions or other XR pipelines may result in incorrect capture behavior."

Reply:
True! My one hesitation is that it _will_ work eventually, and this help box will go out of date at some point in the (hopefully) near future :/

Reply:
I understand your hesitation. The helpbox is already there and suggesting a 2019 version that doesn't work. When I checked with @JBrentJ, we still didn't have a commitment on when a fix will be in from Unity.

Reply:
>we still didn't have a commitment on when a fix will be in from Unity.

I think we can relay that information though, something like "this didn't work on XR SDK when we shipped this MRTK release. see this doc page for the latest information". We can rev docs pages out of band with MRTK releases. I don't want developers to think this is the most up to date information when it might go stale in the next couple of weeks.

Reply:
Updated with this feedback
![image](https://user-images.githubusercontent.com/39840334/105925988-164f3e00-600f-11eb-998f-db32f344bdd2.png)

Reply:
@RogPodge thanks for the quick turn around.
Nit: small grammar typo. I think it should read, "This did not work on XR SDK..." as opposed to "doesn't".

Reply:
Just missed this I'll sneak it into our core-profile feature improvements :)